### ISSUE TITLE
Error installing minio dependency for Argo chart

### ISSUE THREAD

**Issue by cgorman** [argoproj/argo-helm#455]
**Describe the bug**
Trying to install Argo using helm chart 0.12.1 produces the following error:

```
Error: template: argo/charts/minio/templates/deployment.yaml:198:20: executing "argo/charts/minio/templates/deployment.yaml" at <(not .Values.gcsgateway.enabled) (not .Values.azuregateway.enabled) (not .Values.s3gateway.enabled) (not .Values.b2gateway.enabled)>: can't give argument to non-function not .Values.gcsgateway.enabled
```

While looking for [other bug reports](https://github.com/helm/charts/issues/21446), I noticed that they have [moved the Minio charts](https://hub.helm.sh/charts/minio/minio) to helm.min.io. Has that been reflected here? Either way it looks like it was fixed in chart version 5.0.16 so could we get the requirements updated, please?

**To Reproduce**
```
$ helm version
version.BuildInfo{Version:"v3.3.1", GitCommit:"249e5215cde0c3fa72e27eb7a30e8d55c9696144", GitTreeState:"clean", GoVersion:"go1.14.7"}
```
1. Install Argo helm chart with minio.install=true

**Expected behavior**
Argo and minio install correctly.

**Comment by SniperBean** [comment]
I got same issue with helm v2.16.12 when installing Argo Workflow helm chart with minio.install=true

**Comment by thenetengineer** [comment]
Same issue.

**Comment by tsunamishaun** [comment]
I noticed the requirements is pointed at the old repo and version of the minio chart. It has moved to [here](https://github.com/minio/charts/tree/master/minio) but there also appears to be a bitnami chart that is slightly more up to date. I leave that choice up to the maintainers :)

**Comment by jaawasth** [comment]
any updates here ?

**Comment by tsunamishaun** [comment]
@jaawasth Posted on argo [slack](https://argoproj.slack.com/archives/C8J6SGN12/p1604435599312400), hopefully this will get some 👀

**Comment by jaawasth** [comment]
@seanson , fyi, this is the issue which references the minio chart update that needs to be done with argo-helm

**Comment by jaawasth** [comment]
@jessesuen @benjaminws , can you please help here, helm2 will end support on 12th Nov, 2020

**Comment by jiaranda** [comment]
Please merge this PR :eyes:

**Comment by naveenb30** [comment]
I'm also having same issue with `minio.install=true`

**Comment by mszostok** [comment]
FYI: I tested the `0.16.7` Argo Helm Chart version and the MinIO bug is fixed, it was resolved by #546

**Comment by tsunamishaun** [comment]
Thanks for looking into that @mszostok - I noticed the chart is going to be deprecated next week in favor of the minio operator, this would require a bit of work to modify the chart to use a cr - I might open an issue for this. When I do I'll reference this, I think this can be closed now.
